smartsim/log.py

```diff
@@ -129,3 +129,23 @@ def log_to_file(filename: str, log_level: str = "debug") -> None:
         filename, "w+", encoding="utf-8"
     )
     coloredlogs.install(stream=stream, logger=logger, level=log_level)
+
+
+def log_to_file_preview(
+    filename: str,
+    logger: logging.Logger,
+) -> logging.Handler:
+    """Installs a filestream handler to the root logger,
```

Comment:
I think this docstring is a little misleading. Since we don't propagate, it's not really accurate to say "installs...to the root logger"